lib/draper/automatic_delegation.rb

```diff
@@ -18,11 +18,21 @@ def respond_to_missing?(method, include_private = false)
       super || delegatable?(method)
     end
 
-    # @private
-    def delegatable?(method)
-      return if private_methods(false).include?(method)
+    # The inherit argument for `private_method_defined?` is supported since Ruby 2.6.
+    if RUBY_VERSION >= "2.6"
+      # @private
+      def delegatable?(method)
+        return if self.class.private_method_defined?(method, false)
 
-      object.respond_to?(method)
+        object.respond_to?(method)
+      end
+    else
+      # @private
+      def delegatable?(method)
+        return if private_methods(false).include?(method)
```

Comment:
BTW, what this line is for in the first place? 🤔

Reply:
It would be great if we could remove it.

Reply:
I think we can't remove this. This is needed to prevent delegating public methods from being overridden by a private in the decorator. 
Please see #849 for details.

Reply:
Oh, sure, thanks for the clarification. I've misread it as `object.private_methods(false).include?(method)`, sorry.